frontend/catalyst/compilation_pipelines.py

```diff
@@ -35,6 +35,7 @@
 )
 
 from catalyst.utils.gen_mlir import inject_functions
+from catalyst.utils import wrapper  # pylint: disable=no-name-in-module
```

Comment:
This `no-name-in-module` error occurs because pylint is run before catalyst is installed. We can move the pylint check to after catalyst is pip installed.